clang/test/SemaSYCL/intel-fpga-local.cpp

```diff
@@ -298,10 +298,19 @@ void diagnostics()
   [[intel::merge("mrg4", "depths")]] unsigned int mrg_invalid_arg[4];
 
   // Checking of different argument values.
-  //expected-warning@+2{{attribute 'merge' is already applied}}
-  [[intel::merge("mrg4", "depth")]]
+  //expected-warning@+2{{attribute 'merge' is already applied with different arguments}}
+  [[intel::merge("mrg4", "depth")]] // expected-note {{previous attribute is here}}
   [[intel::merge("mrg5", "width")]] unsigned int mrg_mrg[4];
 
+  // Checking of duplicate argument values.
+  // No diagnostic is emitted because the arguments match.
+  //CHECK: VarDecl{{.*}}mrg_mrg6
```

Comment:
Isn't this a diagnostics test? I don't think you need to check AST here again

Reply:
My mistake. Thanks @elizabethandrews for reviews and catching it. I have removed it.